### PR TITLE
Add Periscope to team social media

### PR DESCRIPTION
**Pull request by technonerdz**
This adds periscope support to team social media!

### CODE REVIEW

File: tests/test_media_url_parse.py [hunk 1]
```diff
@@ -106,6 +106,14 @@ def test_instagram_profile_parse(self):
         self.assertEqual(result['foreign_key'], '4hteamneutrino')
         self.assertEqual(result['site_name'], MediaType.type_names[MediaType.INSTAGRAM_PROFILE])
         self.assertEqual(result['profile_url'], 'https://www.instagram.com/4hteamneutrino')
+        
+        def test_periscope_profile_parse(self):
```
Comment (phil-lopreiato) [inline tests/test_media_url_parse.py]: Spacing is wrong here

**Comment by technonerdz** [comment]
I figured out why the test was failing! I will try to run it on a developer server tomorrow!

**Comment by technonerdz** [comment]
When I run a dev server. The site looks like that. Do you know how to fix this? 
ss 2016-10-17 at 07 50 15

**Comment by phil-lopreiato** [comment]
Run `paver make`

**Comment by technonerdz** [comment]
I did that before and this is what it does:
ss 2016-10-17 at 07 57 01

**Comment by phil-lopreiato** [comment]
```
sh: 1: java: not found
```

Is `java` installed?

**Comment by technonerdz** [comment]
I did it!!! here are the screenshots:

Review flow:
ss 2016-10-18 at 05 11 58

On Team page:
ss 2016-10-18 at 05 33 20

Do you think I should add Twitch profiles?

**Pull request merged**